### PR TITLE
refactor: improve debug message retrieving local presets

### PR DESCRIPTION
**Pull request by hackardoX**
Hello,
I tried to use renovate in my environment. However, I am still having some issues trying to configure the onboardConfig with a local extended preset. I continue to receive error 404 even if my repository renovate-config exists, as well as my default.json. So, I thought to clarify the error displayed to add the URL used to retrieve the file.

### CODE REVIEW

File: lib/config/presets/gitlab/index.ts [hunk 1]
```diff
@@ -32,21 +32,21 @@ export async function fetchJSONFile(
   fileName: string,
   endpoint: string
 ): Promise<Preset> {
+  const urlEncodedRepo = encodeURIComponent(repo);
+  const urlEncodedPkgName = encodeURIComponent(fileName);
+  const defaultBranchName = await getDefaultBranchName(
+    urlEncodedRepo,
+    endpoint
+  );
```
Comment (viceice) [inline lib/config/presets/gitlab/index.ts:40]: This can fail and should be moved back to try catch

**Review by viceice — COMMENTED** [review]
can you give us some more context? maybe your bot doesn't have access? please file a new issue [here](https://github.com/renovatebot/config-help/issues/new/choose)

File: lib/config/presets/gitlab/index.ts (hunk 1 above)
Comment (andrea11) [reply lib/config/presets/gitlab/index.ts]: Thank you @viceice for the quick reply. I would open an issue, but I have already a look and I did not find any problem. 
In any case, what do you think about the improvement of logging?

**Comment by viceice** [comment]
I'm generally ok with this, but it won't solve your issue, see config-help for more explanation.

ref https://github.com/renovatebot/config-help/issues/941

**Review by viceice — APPROVED** [review]
lets add those urls to the log message

**Pull request merged**